src/libraries/System.Security.Cryptography/src/System/Security/Cryptography/Pbkdf2Implementation.Windows.cs

```diff
@@ -60,8 +60,7 @@ internal static partial class Pbkdf2Implementation
             if (password.IsEmpty)
             {
                 // CNG won't accept a null pointer for the password.
-                symmetricKeyMaterial = stackalloc byte[1];
-                symmetricKeyMaterialLength = 0;
```

Comment:
@stephentoub - this change is causing build errors:

```
D:\a\_work\1\s\src\libraries\System.Security.Cryptography\src\System\Security\Cryptography\Pbkdf2Implementation.Windows.cs(127,25): error CS0165: Use of unassigned local variable 'symmetricKeyMaterialLength' [D:\a\_work\1\s\src\libraries\System.Security.Cryptography\src\System.Security.Cryptography.csproj::TargetFramework=net9.0-windows]

```
https://dev.azure.com/dnceng-public/public/_build/results?buildId=747203&view=logs&j=d2b92d58-337d-5541-2ecb-d35f35f29352&t=3bcb3c60-9dac-5f6d-6a96-02987273246b

Reply:
Opened https://github.com/dotnet/runtime/issues/105126